cumulusci/tasks/salesforce/tests/test_activate_flow.py

```diff
@@ -0,0 +1,88 @@
+import unittest as test_activate_flow
+import json
+import responses
+from .util import create_task
+from cumulusci.tasks.salesforce.activate_flow import ActivateFlow
+
+
+class TestActivateFlow(test_activate_flow.TestCase):
+    @responses.activate
+    def test_half_activate_flow_processes(self):
```

Comment:
Does "half" here mean we're looking at the case where half (1 of 2) of the supplied Developer Names is actually found in the org?

The test looks good, but it raises an important question - should this situation be an error? That is, should we either (a) warn the user or (b) fail the build if a Flow we're asked to activate does not exist? 

At minimum, let's add a `self.logger.warn()` call in the main task. If the count of returned records isn't equal to the count of developer names, we can warn the user that we expected (X, Y, Z) and only got back (X, Y).